packages/@aws-cdk/aws-rds/lib/instance.ts

```diff
@@ -0,0 +1,938 @@
+      resources: [
+        this.node.stack.formatArn({
+          service: 'rds',
+          resource: 'db',
+          sep: ':',
+          resourceName: this.instanceIdentifier
+        })
+      ]
+    });
+    rule.addTarget(target);
+    return rule;
+  }
+
+  /**
+   * Renders the secret attachment target specifications.
+   */
+  public asSecretAttachmentTarget(): secretsmanager.SecretAttachmentTargetProps {
+    return {
+      targetId: this.instanceIdentifier,
+      targetType: secretsmanager.AttachmentTargetType.Instance
+    };
+  }
+}
+
+/**
+ * The engine for the database instance.
+ */
+export enum DatabaseInstanceEngine {
+  Aurora = 'aurora',
+  AuroraMysql = 'aurora-mysql',
+  AuroraPostgresql = 'aurora-postgresql',
+  MariaDb = 'mariadb',
+  Mysql = 'mysql',
+  OracleEE = 'oracle-ee',
+  OracleSE2 = 'oracle-se2',
+  OracleSE1 = 'oracle-se1',
+  OracleSE = 'oracle-se',
+  Postgres = 'postgres',
+  SqlServerEE = 'sqlserver-ee',
+  SqlServerSE = 'sqlserver-se',
+  SqlServerEX = 'sqlserver-ex',
+  SqlServerWeb = 'sqlserver-web'
+}
+
+/**
+ * The license model.
+ */
+export enum LicenseModel {
+  /**
+   * License included.
+   */
+  LicenseIncluded = 'license-included',
+
+  /**
+   * Bring your own licencse.
+   */
+  BringYourOwnLicense = 'bring-your-own-license',
+
+  /**
+   * General public license.
+   */
+  GeneralPublicLicense = 'general-public-license'
+}
+
+/**
+ * The processor features.
+ */
+export interface ProcessorFeatures {
+  /**
+   * The number of CPU core.
+   */
+  readonly coreCount?: number;
+
+  /**
+   * The number of threads per core.
+   */
+  readonly threadsPerCore?: number;
+}
+
+/**
+ * The type of storage.
+ */
+export enum StorageType {
+  /**
+   * Standard.
+   */
+  Standard = 'standard',
+
+  /**
+   * General purpose (SSD).
+   */
+  GP2 = 'gp2',
+
+  /**
+   * Provisioned IOPS (SSD).
+   */
+  IO1 = 'io1'
+}
+
+/**
+ * The retention period for Performance Insight.
+ */
+export enum PerformanceInsightRetentionPeriod {
+  /**
+   * Default retention period of 7 days.
+   */
+  Default = 7,
+
+  /**
+   * Long term retention period of 2 years.
+   */
+  LongTerm = 731
+}
+
+/**
+ * Construction properties for a DatabaseInstanceNew
+ */
+export interface DatabaseInstanceNewProps {
+  /**
+   * The name of the compute and memory capacity classes.
+   */
+  readonly instanceClass: ec2.InstanceType;
+
+  /**
+   * Specifies if the database instance is a multiple Availability Zone deployment.
+   *
+   * @default false
+   */
+  readonly multiAz?: boolean;
+
+  /**
+   * The name of the Availability Zone where the DB instance will be located.
+   *
+   * @default no preference
+   */
+  readonly availabilityZone?: string;
+
+  /**
+   * The storage type.
+   *
+   * @default GP2
+   */
+  readonly storageType?: StorageType;
+
+  /**
+   * The number of I/O operations per second (IOPS) that the database provisions.
+   * The value must be equal to or greater than 1000.
+   *
+   * @default no iops
```

Comment:
What does no IOPs mean?

Reply:
This is actually setting *provisioned* IOPS. The default behavior is you don't get *provisioned* (aka guaranteed) IOPS... This can incur additional cost when enabled.

So yeah, I guess the phrasing can be improve to this effect. I think copying the documentation from RDS/CloudFormation should be best.